config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
```

Comment:
* As mentioned by the previous CR the cors error still persist
* HINTS 💭 kindly uncomment lines 8 to 15 of the cors.rb and add the URL of your frontend app to origins that should solve your cors errors 
* Also i would suggest you create 5 random messages   
![image](https://user-images.githubusercontent.com/52497246/228450588-a31a503d-3d70-42be-aa38-472d65147427.png)